fe/fe-core/src/main/java/org/apache/doris/common/proc/PartitionsProcDir.java

```diff
@@ -68,7 +68,7 @@
 public class PartitionsProcDir implements ProcDirInterface {
     public static final ImmutableList<String> TITLE_NAMES = new ImmutableList.Builder<String>()
             .add("PartitionId").add("PartitionName")
-            .add("VisibleVersion").add("VisibleVersionTime")
+            .add("VisibleVersion").add("VisibleVersionTime").add("CommittedVersion")
```

Comment:
It is better add it as the last column.

Then old tools do not need to change, which used `show partitions`.

Reply:
tool should not use postition to get field.  mysql client can get field name from response.   put visiable version and commit version together for human friendly readable.